src/app/security-list/security-list.component.html

```diff
@@ -1 +1,78 @@
-<p>security-list works!</p>
+<div class="content">
+  <div class="security-title">TITLE</div>
+
+  <div class="table-wrapper">
+    <div class="security-tabs">
+      <div class="tab active-tab">
+        <div class="tab-content">Actions</div>
+      </div>
+      <div class="tab">
+        <div class="tab-content">Future</div>
+      </div>
+      <div class="tab">
+        <div class="tab-content">Forex</div>
+      </div>
+    </div>
+
+    <div class="table-content">
+      <div class="filter-row">
+        <div class="left-row">
+          <div class="filter-dropdown">
+            <select class="security-filter-select">
+              <option>Show all</option>
+            </select>
+          </div>
+          <div class="filter-input">
+            <input
+              class="security-filter-input"
+              placeholder="Search by email"
+            />
+          </div>
+          <div class="filter-input">
+            <input class="security-filter-input" placeholder="By name" />
+          </div>
+          <div class="filter-input">
+            <input class="security-filter-input" placeholder="By surname" />
+          </div>
+          <div class="search-icon">
+            <span class="material-symbols-outlined"> search </span>
+          </div>
+        </div>
+
+        <div class="right-row">
+          <div class="add-new-btn">ADD NEW USER</div>
+        </div>
+      </div>
+
+      <div class="security-table-wrapper">
+        <div class="wrapper-header">Users</div>
+        <table class="user-table" cellpadding="2px" cellspacing="0">
+          <thead class="table-head-custom">
+            <tr class="tr-rows">
+              <th class="table-head">NAME</th>
+              <th class="table-head">EMAIL</th>
+              <th class="table-head">JMBG</th>
+              <th class="table-head">POSITION</th>
+              <th class="table-head">PHONE NUMBER</th>
+              <th class="table-head">ACTIVITY</th>
+              <th class="table-head"></th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr *ngFor="let user of users" class="tr-rows">
+              <td>{{ user.name }}</td>
+              <td>{{ user.email }}</td>
+              <td>{{ user.jmbg }}</td>
+              <td>{{ user.position }}</td>
+              <td>{{ user.phone }}</td>
+              <td>{{ user.activity }}</td>
+              <td>
+                <span class="material-symbols-outlined"> more_vert </span>
+              </td>
+            </tr>
```

Comment:
Ova tabela je za user-a, potrebno je da u njoj prikazujemo hartije od vrednosti a ne user-e. Andreja nije imala vremena da napravi dve razlicite tabele za user-e i hartiju od vrednosti pa je sve napravila na jednoj. Treba nam ista da tabela samo sa ovim podacima. Prilikom prikaza hartije treba prikazati osnovne podatke o hartiji, kao što su Ticker, Price, Change, Volume i Initial_Margin_Cost.